### PR TITLE
feat(design-system): theme Storybook, add Colour foundation (and story)

### PR DESCRIPTION
**Pull request by karrui**
## Problem

This PR builds on the Typography PR to add the full Colours foundation to the design system, alongside a Story.

Pseudo-state addon was also causing insane lag in Storybook, eventually causing a crash. This package has also been removed. As such, the hover/focus stories cannot be snapshot-tested and must be manually tested.

Related to #1903 

## Solution

**Features**:

- feat(storybook): update theme of Storybook for Inter font
- feat(design-system): add full design system colors to theme
- feat(story): add Colours foundation story

**Fixes**:
- Remove `storybook-addon-pseudo-states` package

### CODE REVIEW

File: frontend/.storybook/foundations/Colours/Colours.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks'
```
Comment (mantariksh) [inline frontend/.storybook/foundations/Colours/Colours.stories.mdx:1]: any reason why we're using a mix of story formats?

**Review by mantariksh — APPROVED** [review]
assuming most of this is boilerplate

**Comment by karrui** [comment]
The documentation page looks much better with the MDX format (and much easier for designers to update when needed).

Will probably use MDX for all foundation type components, and CSF for components themselves.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: frontend/.storybook/foundations/Colours/Colours.stories.mdx (hunk 1 above)
Comment (karrui) [reply frontend/.storybook/foundations/Colours/Colours.stories.mdx:1]: The documentation page looks much better with the MDX format (and much easier for designers to update when needed).

Will probably use MDX for all foundation type components, and CSF for components themselves.

**Pull request merged**